content/docs/reference/php-reference.md

```diff
@@ -9,49 +9,147 @@ menu:
-version, see the [release
-notes](https://github.com/paketo-buildpacks/php/releases).
+version, see the [release notes][bp/php].
 
-## Buildpack-Set Environment Variables
+## Behavior
+The Paketo PHP Buildpack is a [composite buildpack][paketo/composite-buildpack]
+designed to build applications written in PHP.
 
-The PHP CNB sets a number of environment variables during the `build` and
-`launch` phases of the app lifecycle. The sections below describe each
-environment variable and its impact on your app.
+### Package Management
 
-### APP_ROOT
+With the PHP CNB, the only option for package management is
+[Composer][external/composer].
 
-* Set by: `httpd` buildpack
-* Phases: `launch`
-* Value: path of app source
+The [Composer buildpack][bp/composer] will install `composer` as the package manager, and if a PHP
+application contains a `composer.json` (and preferably a `composer.lock` file) packages
+are installed by running `composer install` by the [Composer Install
+buildpack][bp/composer-install]. Packages installed will be available in a
+`vendor` directory in the application source directory, and will be cached for
+rebuilds.
 
-### SERVER_ROOT
+### Running With Webservers
+The buildpacks within the PHP CNB will configure basic configuration for
+working with FPM, HTTPD, Nginx, or the built-in PHP webserver. Web server
+choice is set by the user via environment variable (see PHP How To
+documentation), and user-provided configurations will be considered as outlined
+below.
 
-* Set by: `httpd` buildpack
-* Phases: `launch`
-* Value: path of the httpd installation
+##  Software Bill of Materials
+The PHP buildpack supports the full [software bill of materials]({{< ref
+"docs/concepts/sbom" >}}) (SBOM) in [Syft][format/syft],
+[CycloneDX][format/cyclonedx], [SPDX][format/spdx], and
+[Paketo-specific][format/paketo] formats.
 
-### MIBDIRS
+Apps built with the buildpack that use Composer as a a package manager contain
+SBOM entries that provide a full picture of the packages on the final app image
+that get installed.  Additionally, there are entries for the PHP version used,
+the Composer version used. Check out the [Access the Software Bill of Materials
+guide]({{< ref "docs/howto/sbom" >}}) for more information about how to
+retrieve the SBOM for your app image.
 
-* Set by: `php-dist` buildpack
-* Phases: `build` and `launch`
-* Value: See [php documentation](https://www.php.net/manual/en/snmp.installation.php)
+
+## Buildpack Configurations
+
+The buildpacks in the PHP language family set default configurations, as well
+as include configurations from user-provided sources. The sections below
+outline the various configurations the buildpacks provide during the build process, and
+the various locations that will also be considered as configuration sources.
+
+### [PHP Distribution Buildpack][bp/php-dist] Configuration
+1. The [default PHP `ini` file][bp/php-dist-ini] found from the PHP
```

Comment:
What is this list enumerating? A tiny executive summary at the start of this section would help clarify.

Reply:
Is this list in order of precedence? In chronology?